packages/fluentui/react-builder/src/components/Designer.tsx

```diff
@@ -323,17 +323,14 @@ export const Designer: React.FunctionComponent = () => {
     },
   };
 
-  const handleKeyDown = React.useCallback(
-    (e: KeyboardEvent) => {
-      let command = '';
-      command += e.altKey ? 'Alt+' : '';
-      command += e.ctrlKey || e.metaKey ? 'Ctrl+' : '';
-      command += e.shiftKey ? 'Shift+' : '';
-      command += e.key;
-      hotkeys.hasOwnProperty(command) && hotkeys[command]();
-    },
-    [hotkeys],
```

Comment:
An improvement to the hooks lint rule pointed out that since the `hotkeys` object is re-created on every render, this `useCallback` will run on every render too, so there's no point in having it (and I couldn't see a straightforward way to avoid `hotkeys` being re-created)